sdk-setup/src/sdk-manage

```diff
@@ -1423,7 +1423,7 @@ synchronise_target() {
         maybe_times=
     fi
 
-    echo >&2 "Synchronising target to host"
+    echo >&2 "Synchronising target to host..."
```

Comment:
Have you considered adding --info=progress2 to rsync parameters?

Reply:
That would be nice but unfortunately it does not print nicely with output not directed into terminal.